.changeset/dirty-carrots-obey.md

```diff
@@ -0,0 +1,44 @@
+---
+'@leafygreen-ui/icon': minor
+'@leafygreen-ui/typography': patch
```

Comment:
Nit: break out into two changesets so the icon information isn't in the typography changelog and vice versa

Reply:
Good point! Done

Reply:
Thanks!